code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -106,6 +110,10 @@
 	var/fortify_armor = 0
 	var/crest_defense_slowdown = 0
 
+	// *** Crusher Abilities *** //
+	var/stomp_damage = 0
+	var/crest_toss_distance = 0
+
```

Comment:
All of these `var/xxxxx` need `///` auto doc's explaining what they represent and how they work. See #5236 for examples

Reply:
OKay